docs/upgrade/upgrade-enterprise/index.rst

```diff
@@ -6,42 +6,44 @@ Upgrade Scylla Enterprise
    :hidden:
    :titlesonly:
 
-   Scylla Enterprise 2022 <upgrade-guide-from-2022.x.y-to-2022.x.z/index>
-   Scylla Enterprise 2021 <upgrade-guide-from-2021.x.y-to-2021.x.z/index>
-   Scylla Enterprise 2020 <upgrade-guide-from-2020.x.y-to-2020.x.z/index>
-   Scylla Enterprise 2019 <upgrade-guide-from-2019.x.y-to-2019.x.z/index>
-   Scylla Enterprise 2018 <upgrade-guide-from-2018.x.y-to-2018.x.z/index>
-   Scylla Enterprise 2017 <upgrade-guide-from-2017.x.y-to-2017.x.z/index>
-   Scylla Enterprise 2021.1 to Scylla Enterprise 2022.1 <upgrade-guide-from-2021.1-to-2022.1/index>
-   Scylla Enterprise 2020.1 to Scylla Enterprise 2021.1 <upgrade-guide-from-2020.1-to-2021.1/index>
-   Scylla Enterprise 2019.1 to Scylla Enterprise 2020.1 <upgrade-guide-from-2019.1-to-2020.1/index>
-   Scylla Enterprise 2018.1 to Scylla Enterprise 2019.1 <upgrade-guide-from-2018.1-to-2019.1/index>
-   Scylla Enterprise 2017.1 to Scylla Enterprise 2018.1 <upgrade-guide-from-2017.1-to-2018.1/index>
+   ScyllaDB Enterprise 2022 <upgrade-guide-from-2022.x.y-to-2022.x.z/index>
+   ScyllaDB Enterprise 2021 <upgrade-guide-from-2021.x.y-to-2021.x.z/index>
+   ScyllaDB Enterprise 2020 <upgrade-guide-from-2020.x.y-to-2020.x.z/index>
+   ScyllaDB Enterprise 2019 <upgrade-guide-from-2019.x.y-to-2019.x.z/index>
+   ScyllaDB Enterprise 2018 <upgrade-guide-from-2018.x.y-to-2018.x.z/index>
+   ScyllaDB Enterprise 2017 <upgrade-guide-from-2017.x.y-to-2017.x.z/index>
+   ScyllaDB Enterprise 2022.1 to Scylla Enterprise 2022.2 <upgrade-guide-from-2022.1-to-2022.2/index>
+   ScyllaDB Enterprise 2021.1 to Scylla Enterprise 2022.1 <upgrade-guide-from-2021.1-to-2022.1/index>
+   ScyllaDB Enterprise 2020.1 to Scylla Enterprise 2021.1 <upgrade-guide-from-2020.1-to-2021.1/index>
+   ScyllaDB Enterprise 2019.1 to Scylla Enterprise 2020.1 <upgrade-guide-from-2019.1-to-2020.1/index>
+   ScyllaDB Enterprise 2018.1 to Scylla Enterprise 2019.1 <upgrade-guide-from-2018.1-to-2019.1/index>
+   ScyllaDB Enterprise 2017.1 to Scylla Enterprise 2018.1 <upgrade-guide-from-2017.1-to-2018.1/index>
```

Comment:
are these still supported?
AFAIU they are not, so if they are not, we should remove from here